.github/workflows/ci.yaml

```diff
@@ -12,6 +12,7 @@ concurrency:
 
 env:
   SEGMENT_DOWNLOAD_TIMEOUT_MINS: 2 # Fail cache download after 2 minutes.
+  RUST_VERSION: '1.85'
```

Comment:
Could you also do the same for go version? 🙏🏻  Or should we open separate PR for that?

Reply:
Let's open a separate PR for that please.